index.js

```diff
@@ -2,42 +2,44 @@ var assert = require('assert')
 var path = require('path')
 var homedir = require('os-homedir')
 var raf = require('random-access-file')
-var thunky = require('thunky')
 var mkdirp = require('mkdirp')
+var thunky = require('thunky')
+var toDiscovery = require('hypercore/lib/hash').discoveryKey
```

Comment:
Also maybe get the discovery key PR merged so I can avoid this? https://github.com/mafintosh/hypercore/pull/80